packages/gasket-plugin-lifecycle/index.js

```diff
@@ -26,8 +26,9 @@ async function resolve(root, name) {
     }
   }
 
+  const reg = /.*^((?!(\.test|\.spec).js).)*\.js$/;
```

Comment:
Should make regex(s) case-insensitive for file names.

Reply:
This appears to be fixed using your changes above.